### DISCUSSION TITLE
What's purpose of zero width character in headless/ui dialog?

### DISCUSSION THREAD

**Discussion by qwe12e** [tailwindlabs/tailwindcss#7670] [Help] [answered]
In this [dialog documentation](https://headlessui.dev/vue/dialog) example

I saw this lines and was wondering what is the purpose of this? I couldn't find any comments on about it.
![image](https://user-images.githubusercontent.com/13543079/155800790-f02096dc-0f4b-40df-aa24-25741d5b6678.png)

**Comment by adamwathan** [comment, marked as answer]
Hey! It's a weird little trick to make sure that the modal is centered if it's shorter than the viewport, but if it's taller then the modal backdrop area becomes scrollable.

**Reply by qwe12e** [reply]
Wow that's a neat hack! Thanks for clarifying.
